src/assets/css/site.css

```diff
@@ -5,6 +5,10 @@ html {
     height: 100%;
 }
 
+body.withSectionTabs {
+    overflow-y: scroll;
```

Comment:
Do we want to force the scrollbars to display even if they aren't needed?

Reply:
Crap, I left it, good catch.
Can you push to the branch and remove the "Fix horizontal hop" commit? (Latest one).

I can't do it for another 13-14 hours.

Reply:
Removed.